packages/staking-api-js/client.ts

```diff
@@ -170,6 +171,18 @@ export class SessionStakingClient {
     return await this.request<BlsRewardsResponse>(options);
   }
 
+  public async getRewardsInfoV1({
+    address,
+  }: {
+    address: string;
+  }): Promise<StakingBackendResponse<V1BlsRewardsResponse>> {
+    const options: RequestOptions = {
+      endpoint: `/rewardsv1/${address}`,
```

Comment:
I had a look at the staking backend pr and didn't see this endpoint. Confirming it is not supports to be `/rewards/v1`?

Reply:
This endpoint is an nginx rewrite, wont appear in any backend pr